package.json

```diff
@@ -33,7 +32,7 @@
     "prepublishOnly": "npm run compile && npm run browser-compile"
   },
   "peerDependencies": {
-    "graphql": "^0.10.0 || ^0.11.0"
+    "graphql": "^0.10.0 || ^0.11.0 || ^0.12.0"
```

Comment:
when i was more active we tried to always be comptiable with latest 2 graphql versions
if thats still the case we need to remove 10,
you are already cleaning the quirks we had to do because of version 10=>11 incomptabilty...

Reply:
As `graphql-subscriptions` still supports 0.10 to 0.12, I'm inclined to just add the additional version for now.